solidity/dashboard/src/pages/ResourcesPage.jsx

```diff
@@ -3,21 +3,33 @@ import PageWrapper from "../components/PageWrapper"
 import DocumentationSection from "../components/resources/DocumentationSection"
 import TerminologyDataTable from "../components/resources/TerminologyDataTable"
 import DelegationDiagram from "../components/resources/DelegationDiagram"
-import Navigation from "../components/resources/Navigation"
 
-const ResourcesPage = () => {
-  return (
-    <PageWrapper title="Resources">
-      <div className="resources-page-wrapper">
-        <div>
-          <DocumentationSection />
-          <DelegationDiagram />
-          <TerminologyDataTable />
-        </div>
-        <Navigation />
-      </div>
-    </PageWrapper>
-  )
+const ResourcesPage = ({ title, routes }) => {
+  return <PageWrapper title={title} routes={routes} />
+}
+
+DocumentationSection.route = {
+  title: "Documentation",
+  path: "/resources/documentation",
+  exact: true,
+}
+
+DelegationDiagram.route = {
+  title: "Diagrams",
+  path: "/resources/diagram",
+  exact: true,
+}
+
+TerminologyDataTable.route = {
+  title: "Quick Terminology",
+  path: "/resources/quick-terminology",
+  exact: true,
```

Comment:
I wonder if `exact` can be inferred---by which I mean, if `pages` exists, then `exact: false`, if `pages` does not exist, then `exact: true`, without having to explicitly define it.

Obviously non-blocking.

Reply:
Hmm, I see your point but on the other hand, explicitly defined attributes seem better(to me) than this kind of magic.